### ISSUE TITLE
Insufficient colour contrast on Beta text

### ISSUE THREAD

**Issue by macameron63** [AusDTO/citizenship-appointment-client#1]
The colour contrast ratio (3.08:1) on the Beta-tag is insufficient (less than 4.5:1) for the current font size.

![dto beta colour contrast issue](https://cloud.githubusercontent.com/assets/17154976/14479278/39298fb0-0160-11e6-98e7-459b212befd9.jpg)
![dto beta colour contrast css](https://cloud.githubusercontent.com/assets/17154976/14479275/2c923a36-0160-11e6-8d04-517dae6eebc7.png)

## Suggested Resolution
Changing the color of the text (BETA) from white to black will provide a contrast ratio greater than 4.5:1 (Ref Situation A) alternatively adding CSS font-size = 120% to the class (e.g. Beta-tag) will increase the existing bolded font size to 14 points meaning 3.08:1 is sufficient (Ref Situation B)

### Techniques and Failures for Success Criterion 1.4.3 - Contrast (Minimum) 
**Situation A: text is less than 18 point if not bold and less than 14 point if bold**
[G18: Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text ](http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20160317/G18)
**Situation B: text is at least 18 point if not bold and at least 14 point if bold**
[G145: Ensuring that a contrast ratio of at least 3:1 exists between text (and images of text) and background behind the text ](http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20160317/G145)

**Comment by emsiow** [comment]
Fixed in commit [60b05a4](https://github.com/AusDTO/citizenship-appointment-client/commit/60b05a469b03ef6f40ab356f5105adb100891c57)
